integration-tests/metrics-init.patch

```diff
@@ -10,6 +10,19 @@
      protected TimeUnit parseTimeUnit(String value, TimeUnit defaultValue) {
          try {
              return TimeUnit.valueOf(String.valueOf(value).toUpperCase(Locale.US));
+--- a/metrics-jvm/src/test/java/io/dropwizard/metrics5/jvm/ThreadDumpTest.java
++++ b/metrics-jvm/src/test/java/io/dropwizard/metrics5/jvm/ThreadDumpTest.java
+@@ -37,6 +37,10 @@ class ThreadDumpTest {
+         });
+     }
+ 
++    // XXX: The Refaster `StringFormatted` rule doesn't properly wrap the string concatenation,
++    // causing only the final `%n` occurrence to be replaced with a system line separator.
++    // See https://github.com/google/error-prone/issues/4866.
++    @SuppressWarnings("StringFormatted")
```

Comment:
This resolves the issue. Let's follow up with a PR that adds possibly-redundant parentheses, and then depend on [`UnnecessaryParentheses`](https://errorprone.info/bugpattern/UnnecessaryParentheses) to remove the parentheses if indeed redundant. This will require an extra pass, and ideally we fix google/error-prone#4866, but let's unblock a possible release first.

NB: Other modified integration test code is impacted by the same issue; it just doesn't cause a test failure.

Reply:
I filed #1982.